### DISCUSSION TITLE
part-ii/2017-130/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#377] [Question comments]
Part II 2017 Riemann Surfaces: Paper 1, Section II, F 

https://questions.tripos.org/part-ii/2017-130/

**Comment by artie2000** [comment]
First part is on 2021 example sheets.

R-H is bookwork.

<details>
<summary>Last part (a bit fiddly):</summary>

From R-H, there are two ramification points $z_1$ and $z_2$, each of degree 2, mapping to distinct branch points $w_1,$ and $w_2$. Choose Mobius maps so that:

$T(0)=z_1$, $T(\infty)=z_2$, $S(w_1)=0$, $S(w_2)=\infty$.

Because ramification indices are multiplicative and Mobius maps have degree 1, $g=SfT$ has two ramification points 0 and $\infty$, both of which it fixes. Repeating (parts of) the proof in the first part, $g$ is a polynomial.

Since the degree of a polynomial is equal to its degree as a map of spheres, $g$ is a quadratic; since the only root is 0, $g(z)=z^2$.

</details>
